clients/da-vinci-client/src/main/java/com/linkedin/davinci/ingestion/isolated/IsolatedIngestionServerHandler.java

```diff
@@ -371,4 +370,15 @@ private IngestionAction getIngestionActionFromRequest(HttpRequest req) {
           e);
     }
   }
+
+  protected void validateAndExecuteCommand(IngestionTaskReport report, Runnable commandRunnable) {
+    String topic = report.topicName.toString();
+    int partition = report.partitionId;
+    if (isolatedIngestionServer.isPartitionSubscribed(topic, partition)) {
+      commandRunnable.run();
+    } else {
+      report.isPositive = false;
```

Comment:
Can you add some comments here to empathize  a bit about the reason to reject as you have done in the description part, so that reader can pay a bit more attention.